src/vs/platform/terminal/test/common/capabilities/commandDetection/promptInputModel.test.ts

```diff
@@ -693,11 +693,11 @@ suite('PromptInputModel', () => {
 			await assertPromptInput(`ech|`);
 
 			await writePromise('o ');
-			await assertPromptInput(`echo |`);
+			await assertPromptInput(`ech\no|`);
```

Comment:
This test suite is ambiguous, I think we need 2 different suites: `multi-line wrapped` and `multi-line wrapped (no continuation prompt)`.

For `multi-line` and `multi-line wrapped` suites we can then call `setContinuationPrompt('∙ ');` inside the suite setup.